Tests/PointFreeTests/ApiTests.swift

```diff
@@ -24,7 +24,9 @@ final class ApiTests: TestCase {
       |> siteMiddleware
       |> Prelude.perform
 
+    #if !os(Linux)
     assertSnapshot(matching: conn, as: .conn)
+    #endif
```

Comment:
I'm getting this weird diff in linux for this test:

```diff
   "id" : 1,
   "image" : "",
   "length" : 1380,
   "previousEpisodesInCollection" : [
-
+    ¬
   ],
   "publishedAt" : 519652800,
   "references" : [
-
+    ¬
   ],
   "sequence" : 1,
   "subscriberOnly" : false,
   "title" : "Type-Safe HTML in Swift",
```

linux seems to add spaces to the empty array??!!